django/core/servers/basehttp.py

```diff
@@ -70,6 +70,8 @@ class WSGIServer(simple_server.WSGIServer, object):
     def __init__(self, *args, **kwargs):
         if kwargs.pop('ipv6', False):
             self.address_family = socket.AF_INET6
+        if not kwargs.pop('allow_reuse_address', True):
+            self.allow_reuse_address = False
```

Comment:
How about `allow_reuse_address = kwargs.pop('allow_reuse_address', True)` (no if statement) or adding `allow_reuse_address=True` to the signature?